Add submitted lighting talk proposal for the NumFOCUS summit 2023 (#49)
scikit-image/skimage-archive: conferences/Lighting_talk_NumFOCUS_summit_2023_Amsterdam.md

# Lightning talk proposal

for the NumFOCUS summit 2023 in Amsterdam.

## Important dates

- CFP opens: Tuesday, August 22nd, 2023, midnight AoE
- CFP closes: Tuesday, September 5th, 2023, midnight AoE
- CFP acceptance notifications: Thursday, September 7th, 2023, midnight AoE
- Deadline to submit a slide deck (if using): Saturday, September 9th, 2023, midnight AoE

## Track

"Roadmap to a Roadmap"

The discussion topics may include (but not limited to):

- Funding strategy
- Community management
- Governance and CoC
- Formation and execution of ‘Processes’

## Speaker

Lars Grüter

Lars is currently working as a maintainer for the image processing library scikit-image. With an education in electrical engineering and a focus on health and sensor technologies, he has been working as a research assistant on adaptive ultrasound imaging at the TU Dresden. As a student, he started contributing to the scientific Python ecosystem and discovered his interest in signal processing, Linux, and especially Python’s scientific ecosystem. He enjoys fine-tuning algorithms and discussing the finer points of designing an API.

## Proposal summary

A sustainable API transition for scikit-image

> Like many open source projects, scikit-image has grown organically over the past 15+ years, with functionality being added by a broad community of contributors from different backgrounds. This has resulted in various parts of the API being inconsistent and certain early API choices turning out to be more confusing than helpful. Unfortunately, addressing these issues is, in many cases, too burdensome, noisy, or even impossible to fix with a standard deprecation cycle.

Reviewer: ```suggestion
Like many open source projects, scikit-image has grown organically over the past 15+ years, with functionality being added by a broad community of contributors from different backgrounds. As a result, there are inconsistencies between various parts of the API, and certain early API choices have turned out to be more confusing than helpful. Unfortunately, addressing these issues is often too burdensome, noisy, or even impossible to handle with a standard deprecation cycle.
```
Minor (feel free to ignore the suggestion).

> Given the accumulation of potential API changes, the project is now exploring technical solutions to realize this API transition without estranging a large part of its user base or placing an unsustainable burden on its maintainers.

Reviewer: ```suggestion
Given the accumulation of potential API changes, we are now exploring technical solutions to carry out this API transition without estranging a large part of our user base or placing an unsustainable burden on our maintainers.
```

> As many of these aspects likely translate to and affect other projects in the ecosystem, we would like to exchange experiences and insights with the wider community. During the talk, we will highlight the problems we are trying to solve, share our insights, and summarize our ongoing discourse. We hope to start a fruitful discussion around this topic.

Reviewer: "discourse?" Not sure I understand here.

Author: I just looked up a synonym for "discussion". My impression was that I can use it that way?
